README.md

```diff
@@ -186,9 +187,37 @@ fmt.Println(formattedStr)
 //   main.readFile | .../example/main.go | 6
 ```
 
-### Writing your own custom format
+### Writing custom output format
 
-`eris` also allows advanced users to write their own custom formats in case the default format doesn't fit your requirements. The [`UnpackedError`](https://godoc.org/github.com/rotisserie/eris#UnpackedError) object provides a convenient and developer friendly way to store and access existing error traces. The `ErrRoot` and `ErrChain` fields correspond to the root error and wrap error chain, respectively. If any other error type is unpacked, it will appear in the `ExternalErr` field. You can access all of the information contained in an error via [`eris.Unpack`](https://godoc.org/github.com/rotisserie/eris#Unpack).
+`eris` also allows advanced users to construct custom error strings or objects in case the default error doesn't fit the requirements. The [`UnpackedError`](https://godoc.org/github.com/rotisserie/eris#UnpackedError) object provides a convenient and developer friendly way to store and access existing error traces. The `ErrRoot` and `ErrChain` fields correspond to the root error and wrap error chain, respectively. If any other error type is unpacked, it will appear in the `ExternalErr` field. You can access all of the information contained in an error via [`eris.Unpack`](https://godoc.org/github.com/rotisserie/eris#Unpack).
+
+```golang
+// get the unpacked error object
+uErr := eris.Unpack(err)
+//send only the root error message to a logging server instead of complete error trace
+sentry.CaptureMessage(uErr.ErrRoot.Msg)
+```
+### Sending error trace to sentry
+
+`eris` supports sending your error traces to [sentry](https://sentry.io/) using the sentry golang [client SDK](https://github.com/getsentry/sentry-go). You can run the example that generated the following output on sentry UI using the command `go run eris/sentry/example.go -dsn=<DSN>` in the rotisserie [examples](https://github.com/rotisserie/examples) repository.
```

Comment:
Capitalize Sentry
change golang -> Go
maybe change `rotisserie [examples]` to `our [examples]`